### ISSUE TITLE
Landscape and portrait

### ISSUE THREAD

**Issue by erperejildo** [allenRoyston/ngCroppie#14]
Can this module works with this modes? I mean, taking the full size if I do zoom out

**Comment by mikkelking** [comment]
Yes is the simple answer, as long as your crop area matches the aspect ratio of the image.

On 24/08/2016 10:05 PM, Daniel wrote:

Can this module works with this modes? I mean, taking the full size if I do zoom out

**Comment by erperejildo** [comment]
http://foodapp-88007.onmodulus.net/photo

Click on the gallery icon and import an image.

I was looking that kind of functionality but with better results. Right now that library is not able to do that, I've changed it a bit.
So yes, the crop area should adapt to the dimensions but only if you do zoom out.
